Formula/picolibc.rb

```diff
@@ -0,0 +1,38 @@
+class Picolibc < Formula
+  desc "C library designed for embedded systems, built for the host machine"
+  homepage "https://keithp.com/picolibc/"
+  url "https://github.com/picolibc/picolibc/releases/download/1.8.1/picolibc-1.8.1.tar.xz"
+  sha256 "f5366cc5b0103769f518c06979be5e600d48d3f35d0cf2fe8a08098fb37447bb"
+  license all_of: [
+    "BSD-3-Clause",
+    "BSD-2-Clause",
+    :public_domain,
+    :cannot_represent, # assorted other licenses
+  ]
+  head "https://github.com/picolibc/picolibc.git", branch: "main"
+
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
```

Comment:
```suggestion
  depends_on "ninja" => :build
  uses_from_macos "llvm"
```
Since you have `fails_with :gcc` below. Not sure if LLVM is only a build dependency or is needed at runtime too.

Reply:
Should only be needed at build time?